lib/process-services/src/lib/task-list/components/task-filters.component.spec.ts

```diff
@@ -351,4 +363,30 @@ describe('TaskFiltersComponent', () => {
         await fixture.whenStable();
         expect(component.currentFilter).toBe(undefined);
     });
+
+    describe('Display Task Filters', () => {
+
+        it('Should be able to display default task filters', async  () => {
+            spyOn(taskFilterService, 'getTaskListFilters').and.returnValue(of([]));
+            const defaultTaskFiltersMock = [
+                { name: 'dafult-my-filter' },
```

Comment:
You have a lot of typos in the word default. dafult => default also on the data-automation-ids etc

Reply:
Updated